hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/snapshot/TestSnapshotDiffManager.java

```diff
@@ -1174,7 +1175,7 @@ public void testListSnapshotDiffJobs(String jobStatus,
     // there should be a response.
     // Otherwise, response list should be empty.
     if (containsJob) {
-      assertTrue(jobList.contains(diffJob));
+      assertThat(jobList).contains(diffJob);
     } else {
       assertTrue(jobList.isEmpty());
```

Comment:
nit: this can be replaced, too.  This way test failure shows list contents if it is found not empty.

```suggestion
      assertThat(jobList).isEmpty();
```